cursorless-talon/src/actions/actions_simple.py

```diff
@@ -46,6 +53,7 @@
 # Don't wait for these actions to finish, usually because they hang on some kind of user interaction
 no_wait_actions = [
     "generateSnippet",
+    "rename",
```

Comment:
Since we are not waiting for the rename action any sleep extension side doesn't really matter. Maybe we should turn this into an dict with the value being the sleep time out that we run talon side for that no waiting actions?

Reply:
Wait so does this mean that in your PR, rename no longer has any sleep?

Reply:
Well it sleeps extension side but since the client is not waiting for it so in practice no.

Reply:
Maybe just have a separate dict mapping from action id to sleep time. Not sure we want to fold it into the no-wait dict

Reply:
Done